src/user/room.ts

```diff
@@ -428,23 +428,31 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = Array.isArray(mention)
+                                    ? [...mention]
+                                    : mention ? [mention] : []
+
+    const mentionAliasPromiseList = await Promise.all(
```

Comment:
Rename `mentionAliasPromiseList` to `mentionAliasList`

Reply:
since below discussion, we should give a better name since that is await. 
However, ```mentionAliasList``` contain ```string | null``` base on ```Promise.all()``` fail will return ```null```. 
Therefore, I suggest name should be ```mentionAliasListResult``` or ```mentionAliasListPromiseResult```, but the second one name too long..